src/boot/boot.c

```diff
@@ -2950,8 +2957,13 @@ static EFI_STATUS run(EFI_HANDLE image) {
                 (void) process_random_seed(root_dir);
 
                 err = image_start(image, entry);
-                if (err != EFI_SUCCESS)
-                        return err;
+                if (err != EFI_SUCCESS) {
+                        log_error_status(err, "Error starting image: %m");
```

Comment:
image_start() already logs when failed. Not necessary to log again here.

Reply:
updated PR to remove this